imos_bgc_db/NRS_StationInfo.schema.yaml

```diff
@@ -0,0 +1,45 @@
+fields:
+  - name: STATIONCODE
+    description: IMOS unique code identifying the station
+    type: string
+    unique: true
+  - name: STATIONNAME
+    description: Station name
+    type: string
+    unique: true
+  - name: NODE
+    description: IMOS Node operating the station
+    type: string
+  - name: STATECODE
+    description: Australian state the station is located in
+    type: string
+  - name: LONGITUDE
+    description: Longitude (decimal degrees East)
+    type: number
+    maximum: 180
+    minimum: -180
```

Comment:
We can also specify [constraints](https://specs.frictionlessdata.io/table-schema/#constraints) like this, but unfortunately `frictionless` doesn't seem to automatically check these when validating.